File: C++/NexWeb_sum_of_digits_of_a_number.cpp
    #include <iostream>  
   
    int main()  
    {  
    int n,sum=0,m;    
    std::cout<<"Enter a number to sum the digits of: ";    
    std::cin>>n;    
    while(n>0)    
    {    
    m=n%10;    
    sum=sum+m;    
    n=n/10;    
    }    
    std::cout<<"Sum is = "<<sum<< std::endl;    
    return 0;  
    }  

File: Go/NexWeb_sum_of_digits_of_a_number.go
package main

import (
	"fmt"
)

func sumDigits(number int) int {
	remainder := 0
	sumResult := 0
	for number != 0 {
		remainder = number % 10
		sumResult += remainder
		number = number / 10
	}
	return sumResult
}
func main() {
	var number int
	fmt.Print("Enter a number:")
	fmt.Scanln(&number)
	fmt.Printf("The addition of %d = %d\n", number, sumDigits(number))

}
